llmfoundry/utils/model_download_utils.py

```diff
@@ -57,6 +64,7 @@ def download_from_hf_hub(
         save_dir (str, optional): The local path to the directory where the model files will be downloaded.
         prefer_safetensors (bool): Whether to prefer Safetensors weights over PyTorch weights if both are
             available. Defaults to True.
+        tokenizers_only (bool): If true, only download tokenzier files.
```

Comment:
```suggestion
        tokenizer_only (bool): If true, only download tokenizer files.
```
Thinking singular makes more sense?

Reply:
ah yes you're right! good catch